packages/backend-defaults/src/entrypoints/urlReader/lib/GerritUrlReader.ts

```diff
@@ -90,8 +90,17 @@ export class GerritUrlReader implements UrlReaderService {
     url: string,
     options?: UrlReaderServiceReadUrlOptions,
   ): Promise<UrlReaderServiceReadUrlResponse> {
-    const apiUrl = getGerritFileContentsApiUrl(this.integration.config, url);
+    let apiUrl = getGerritFileContentsApiUrl(this.integration.config, url);
     let response: Response;
+    const tagIndex = apiUrl.indexOf('tags');
```

Comment:
Why not parse the tag from the url directly using "parseGitilesUrlRef". Then you don't have to write any new url parsing code. 

There's no tests for "readUrl" using tag url?

Reply:
Acknowledged!